OreNoTask/app/helpers/sessions_helper.rb

```diff
@@ -17,6 +17,10 @@ def logged_in?
     !current_user.nil?
   end
 
+  def admin?
+    current_user.privilege == 1
+  end
```

Comment:
ここもEnumの仕様をうまく使うと以下のように書き換えができそうですー
参考）https://api.rubyonrails.org/v5.2.5/classes/ActiveRecord/Enum.html

```suggestion
  def admin?
    current_user.privilege?
  end
```

Reply:
enum使えばcurrent_user.admin? で動きましたが、enumrizeと設定かぶるのでこのままにしたいです

Reply:
承知しました！